application/src/main/java/org/togetherjava/tjbot/CommandReloading.java

```diff
@@ -33,12 +33,13 @@ private CommandReloading() {
      * @param jda the JDA to update commands on
      * @param commandProvider the {@link CommandProvider} to grab commands from
      */
-    public static void reloadCommands(JDA jda, CommandProvider commandProvider) {
+    public static void reloadCommands(final JDA jda, final CommandProvider commandProvider) {
         logger.info("Reloading commands...");
-        List<CommandListUpdateAction> actions = Collections.synchronizedList(new ArrayList<>());
+        // 110 is based on the fact you can have 100 slash commands, and 20 context commands max.
+        List<CommandListUpdateAction> actions = Collections.synchronizedList(new ArrayList<>(120));
```

Comment:
the better tool for documenting this would be a constant with javadoc. `MAX_COMMANDS = 120` and then this into javadoc.

Reply:
funny how my comment isnt even correct

"110 is because 100 + 20"

Reply:
quick maff